main.cpp

```diff
@@ -35,51 +36,51 @@ Sub Object 1: Steering Wheel
     Name 4 things you'll find on the:    Steering Wheel
         1) paddle shifters
         2) 'cruise control' controls
-        3) 
-        4) 
+        3) Speaker volume controls
+        4) Window wiper controls
     Name 2 things you can do with the:   Steering Wheel
         1) adjust cruise control settings.
-        2)
+        2) Steer the car
         
 Sub Object 2: Instrument Cluster
     Name 4 things you'll find on the:   Instrument Cluster
-        1)
-        2)
-        3)
-        4)
+        1)Speedometer
+        2)Gas amount
+        3)Temperature outside
+        4)Current Time
     Name 3 things you can do with the:   Instrument Cluster
-        1)
-        2)
-        3)
+        1)Measure speed
+        2)Check Gas amount
+        3)Check Distance traveled
     
 Sub Object 3: Environment Controls
     Name 3 things you'll find on the:    Environment Controls
-        1)
-        2)
-        3)
+        1)Temperature Control
+        2)Fan level
+        3)Heat/Ac Toggle
     Name 3 things you can do with the:   Environment Controls
-        1)
-        2)
-        3)
+        1)Change temperature
+        2)Change fan level
+        3)Switch between heat and AC
 
 Sub Object 4: Infotainment System
     Name 3 things you'll find on the:    Infotainment System
-        1)
-        2)
-        3)
+        1)Song titles
+        2)Length of song
+        3)GUI to look through music
     Name 3 things you can do with the:   Infotainment System
-        1)
-        2)
-        3)
+        1)Skip song
+        2)Restart Song
+        3)Browse music selection
```

Comment:
I would have also accepted things related to the Navigation System, Radio, and Car safety features and settings (door lock timeouts, etc)